src/api/Nhs.Appointments.Core/IBookingsDocumentStore.cs

```diff
@@ -25,6 +25,12 @@ public interface IRolesStore
     Task<IEnumerable<Role>> GetRoles();
 }
 
+public interface INotificationConfigurationStore
+{
+    Task<NotificationConfiguration> GetNotificationConfiguration(string eventType);
+    Task<NotificationConfiguration> GetNotificationConfigurationForService(string serviceId, string eventType);
```

Comment:
Pernickety comment warning. I would overloaded the method and have the params the other way round for consistency
e.g.
Task<NotificationConfiguration> GetNotificationConfiguration(string eventType);
Task<NotificationConfiguration> GetNotificationConfiguration(string eventType, string serviceId);